cmd/osm-preinstall/osm-preinstall.go

```diff
@@ -98,3 +101,24 @@ func singleMeshOK(clientset kubernetes.Interface, enforceSingleMesh bool) func()
 		return nil
 	}
 }
+
+func namespaceHasNoMesh(clientset kubernetes.Interface, namespace string) func() error {
+	return func() error {
+		deps, err := clientset.AppsV1().Deployments(namespace).List(context.TODO(), metav1.ListOptions{
```

Comment:
Sometimes deps could mean dependencies. I would have spelled it out as `deployments` to avoid ambiguity.

Reply:
I think at least in this case the scope is narrow enough that there isn't too much ambiguity.